massa-final-state/src/final_state.rs

```diff
@@ -98,15 +146,15 @@ impl FinalState {
         // update current slot
         self.slot = slot;
 
-        // apply changes
+        // apply the state changes
+        // unwrap is justified because every error in PoS `apply_changes` is critical
         self.ledger
             .apply_changes(changes.ledger_changes.clone(), self.slot);
         self.async_pool
             .apply_changes_unchecked(&changes.async_pool_changes);
         self.pos_state
             .apply_changes(changes.pos_changes.clone(), self.slot, true)
-            .expect("could not settle slot in final state proof-of-stake");
-        // TODO do not panic above: it might just mean that the lookback cycle is not available
+            .unwrap();
```

Comment:
I don't understand this change (from expect to unwrap)

Reply:
The reason of the expect was initially that an unavailable PoS cycle should cause a new bootstrap attempt instead of a crash, to be implemented in the future